src/app/page.tsx

```diff
@@ -45,12 +45,20 @@ export default function Home() {
       );
 
       if (data) {
-        setResult(data.example);
-        console.log("payload: ", payload, "data: ", data);
+        if (data.status == 200) {
```

Comment:
Can we limit the line length of the request? I think 100 line (not text/char) should be enough?